agent/cmd/root.go

```diff
@@ -40,8 +40,8 @@ to use to perform stack operations.`,
 		zc := zap.NewDevelopmentConfig()
 		zc.DisableCaller = true
 		zc.DisableStacktrace = true
-		if !verbose {
-			zc.Level.SetLevel(zap.InfoLevel)
+		if verbose {
+			zc.Level.SetLevel(zap.DebugLevel)
```

Comment:
The default level is already info, so debug seemed more appropriate for verbose.